expr/functions.go

```diff
@@ -15,9 +15,93 @@ var ErrWrongArgc = errors.New("wrong number of arguments")
 var ErrBadArgument = errors.New("bad argument")
 
 var allFns = map[string]Function{
+	"Math.max":  mathMax,
+	"Math.min":  mathMin,
 	"Math.sqrt": mathSqrt,
 }
 
+func mathMax(args []zngnative.NativeValue) (zngnative.NativeValue, error) {
+	if len(args) == 0 {
+		return zngnative.NativeValue{}, fmt.Errorf("Math.max: %w", ErrWrongArgc)
+	}
+
+	switch args[0].Type.ID() {
+	case zng.IdInt16, zng.IdInt32, zng.IdInt64:
+		ret := args[0].Value.(int64)
+		for _, val := range args[1:] {
+			v, ok := zngnative.CoerceNativeToInt(val)
+			if ok && v > ret {
+				ret = v
+			}
+		}
+		return zngnative.NativeValue{zng.TypeInt64, ret}, nil
```

Comment:
(not feedback on this PR, but seeing this makes me think that `zngnative.Value` might work better...)

Reply:
that's a great idea, I'll do that in a separate PR